site/src/examples/stepped-tracker/Status.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactElement } from "react";
+import { StackLayout } from "@salt-ds/core";
+import { SteppedTracker, TrackerStep, StepLabel } from "@salt-ds/lab";
+
+export const Status = (): ReactElement => {
+  return (
+    <StackLayout
+      direction="column"
+      align="stretch"
+      gap={10}
+      style={{ width: "100%", minWidth: 600, maxWidth: 800, margin: "auto" }}
```

Comment:
Is this layout doing anything?

Reply:
Following the pattern set by the other examples (Basic, etc.) - having an explicit container width is important, otherwise the steps look way too compressed 
<img width="447" alt="Screenshot 2024-07-08 at 15 24 49" src="https://github.com/jpmorganchase/salt-ds/assets/5412181/8d8e3a4d-92da-41df-8f79-a9fe2586d38d">

Reply:
the width is fine but now sure about it being a stack layout with one child?